src/__tests__/browserSuites/ready-from-cache.spec.js

```diff
@@ -503,10 +503,6 @@ export default function (fetchMock, assert) {
     const client = splitio.client();
     const manager = splitio.manager();
 
-    client.once(client.Event.SDK_READY_FROM_CACHE, () => {
```

Comment:
Rather than simply removing asserts, it is better to update them to keep information of the expected behaviour:

```
client.once(client.Event.SDK_READY_FROM_CACHE, () => {
  t.fail('It should not emit SDK_READY_FROM_CACHE because ...');
  t.end();
});
```

Idem for the other removed asserts.

Reply:
With this change, none of the tests below line 472 (/** Fetch specific splits **/) is asserting that the SDK_READY_FROM_CACHE event is emitted.

So, I would create a new test for the following basic case: 
```
  assert.test(t => { // Testing when we start with cached data with split filter, and the same split filter is provided in the config
```
This should emit the event because the config has the same split filter as the one in the cache.